README.md

```diff
@@ -1 +1,25 @@
 # MediaMonks - Frontend Coding Standards
```

Comment:
We might need a section on how to setup the `import` rules in your project, depending on what bundler you're using:
- node (I think works by default)
- js (I think works by default)
- webpack (requires config in your project)
- pure TS (requires config in your project - or could be included in this package if we choose to, is another comment)
- what about parcel/rollup/etc?

If the user doesn't do this, the project will report errors on all imports that are not node/node_modules.

Reply:
FYI; to get this working with ejected CRA, I had to do this:

_config/webpack.config.eslint.js_
```js
// required by CRA 'env' script, is not correctly passed by eslint config
process.env.NODE_ENV = 'development';

const getConfig = require("./webpack.config");

const config = getConfig("development");

module.exports = config;
```

_.eslintrc.js_
```js
module.exports = {
  root: true,
  extends: ...
  plugins: ...
  settings: {
    'import/resolver': {
      webpack: {
        config: 'config/webpack.config.eslint.js',
      },
    },
  },
```

Reply:
Moved to issue #28